### ISSUE TITLE
10-7959a: Introduction 

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#86810]
As a VA.gov user, I want to read about the CHAMPVA claim form so that I know what to expect before I file a claim.

- breadcrumb 
- title and subtitle (CHAMPVA Claim Form...)
- intro copy with link (goes to legacy url for now - will redirect in futurea): https://www.va.gov/COMMUNITYCARE/programs/dependents/champva/champva-claim.asp
- sign in box (sign in button and link for unauth flow)
- respondent burden info
- OMB control #
- privacy act link
- footer

[Figma link](https://www.figma.com/design/Tfhq5h2LwXEeEEtFBAAFOv/CHAMPVA-Claims-(10-7959a)?node-id=961-139667&t=Ko13IszQmJiM9ZRY-0)

Acceptance criteria:

- Given the user is on VA.gov and about to fill out the CHAMPVA claim form
- When the user gets to the intro page
- Then the user can sign in or review the information before they start filling out the form

**Comment by michaelclement** [comment]
Work is done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30622), waiting on:
- [X] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
Screenshots:
| Before | After
| - | - |
|<img width="566" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNmlrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--105330abaff85fa893b2a8c6b36d9d54b74dca46/before.png" alt="before.png" />|<img width="625" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnFrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a9811db56e6bbda95095fe1a80b20f5d81bd284b/after.png" alt="after.png" />|

**Comment by michaelclement** [comment]
@baltes @marywang2 @syd-hoeper This has been merged into staging and should be visible shortly. [Form link here](https://staging.va.gov/ivc-champva/10-7959a)

**Comment by baltes** [comment]
I didn't catch this before closing...should we see this footer?

<img width="1180" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkNyQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2daa48e76b66a1cd5d891f2ba7ada93f318be6a9/image.png" alt="image.png" />
